minicluster/src/main/java/alluxio/underfs/UnderFileSystemCluster.java

```diff
@@ -89,31 +83,8 @@ public static synchronized UnderFileSystemCluster get(String baseDir)
    * @return the {@link UnderFileSystemCluster}
    */
   public static UnderFileSystemCluster getUnderFilesystemCluster(String baseDir) {
-    sUnderFSClass = System.getProperty(INTEGRATION_UFS_PROFILE_KEY);
-
-    if (!StringUtils.isEmpty(sUnderFSClass)) {
-      try {
-        UnderFileSystemCluster ufsCluster =
-            (UnderFileSystemCluster) Class.forName(sUnderFSClass).getConstructor(String.class)
-                .newInstance(baseDir);
-        LOG.info("Initialized ufs cluster {} for integration testing.", sUnderFSClass);
-        return ufsCluster;
-      } catch (Exception e) {
-        LOG.warn("Failed to initialize the ufs cluster {} for integration testing: {}",
-            sUnderFSClass, e.getMessage());
-        throw Throwables.propagate(e);
-      }
-    }
-    LOG.info("Using default {} for integration testing.", LocalFileSystemCluster.class.getName());
-    return new LocalFileSystemCluster(baseDir);
-  }
-
-  /**
-   *
-   * @return the {@link UnderFileSystem} class name
-   */
-  public static synchronized String getUnderFSClass() {
-    return sUnderFSClass;
+    LOG.info("Using default {} for integration testing.", DefaultUnderFileSystemCluster.class.getName());
+    return new DefaultUnderFileSystemCluster(baseDir);
```

Comment:
Sounds good, since it is not designed to be pluggable anymore. Do you want to do this in this PR?